### PR TITLE
add SAS-CBSD registration test 10.3.4.1.2.1

### PR DESCRIPTION
**Pull request by janetliuboston**
Single-Step registration (Cat A CBSD with no existing CBSD ID)

### CODE REVIEW

File: src/harness/testcases/registration_testcase.py [hunk 1]
```diff
@@ -43,6 +43,24 @@ def test_10_3_4_1_1_1(self):
     response = self._sas.Registration(request)['registrationResponse'][0]
     # Check registration response
     self.assertEqual(response['response']['responseCode'], 0)
+    
+  @winnforum_testcase
+  def test_10_3_4_1_2_1(self):
```
Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: I think this is correctly named 10_3_4_1_2_1 Single-Step registration (Cat A with no existing CBSD ID)?

Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: Shall we just switch to using the new names now? I think this one should be WINFF.FT.S.REG.6.

Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Yes, thanks.

**Comment by janetliuboston** [comment]
Updated test case name to test_WINFF_FT_S_REG_6.

**Pull request merged**